src/main/java/gtPlusPlus/xmod/gregtech/common/tileentities/machines/multi/production/GregtechMetaTileEntityTreeFarm.java

```diff
@@ -144,6 +144,23 @@ public boolean isCorrectMachinePart(final ItemStack aStack) {
 		return TreeFarmHelper.isValidForGUI(aStack) && !GT_ModHandler.isElectricItem(aStack) || GT_ModHandler.canUseElectricItem(aStack, 1);
 	}
 
+	/**
+	 * Method used to get the boost based on the ordinal of the saw
+	 * @param sawOrdinal ordinal of the saw
+	 * @return an int corresponding to the boost
+	 */
+	public int getSawBoost(int sawOrdinal){
```

Comment:
Depending on ordinal can be broken when we add more saw types (and it's a bit hard to read corresponding type), so I'd suggest using enum `SAWTOOL` instead